### PR TITLE
[TASK] Implementing feature to turn on and off caching for  /users/{user_id}/tasks/{task_id}

### LINKED ISSUE #278: [TASK] Implementing feature to turn on and off caching for  /users/{user_id}/tasks/{task_id}

### PR DESCRIPTION
**Pull request by jaguiniga**
Added CACHE TOGGLE and got rid of the testing messages.

### CODE REVIEW

**Review by pouln — APPROVED** [review]
Looks good

**Review by chonger878 — APPROVED** [review]
Simple cache toggle method. Not too complicated to follow.

File: functions/HttpTriggerAPIUsersIdTaskId/__init__.py [hunk 1]
```diff
@@ -16,16 +16,11 @@
     password= os.environ['ENV_REDIS_KEY'], 
     ssl=True)
 
+# Global variables for Redis cache toggle and the invalidation of tasks all 
 CACHE_TOGGLE = os.environ["CACHE_TOGGLE"],
```
Comment (ahmadafrasiyab) [inline functions/HttpTriggerAPIUsersIdTaskId/__init__.py:20]: I like the naming scheme being used

**Review by ahmadafrasiyab — APPROVED** [review]
The toggling part is simple. Just wondering if there is a gui too that developers can use to change the toggle or they have to do it manually?

Looks good to me.

**Pull request merged**